### ISSUE TITLE
change installation directory structure for single node deployment

### ISSUE THREAD

**Issue by prasadtalasila** [AutolabJS/AutolabJS#199]
1. create / map all the required files and volumes from /opt/autolabjs/ to containers.
2. Update setup.sh, keys.sh and playbook files (*.yml) appropriately.

**Comment by prasadtalasila** [comment]
The desirable directory structure is as follows.
```shell
/opt/autolabjs
    main_server
    load_balancer
    execution_node (please note the singular notation)
    gitlab
    mysql
    deploy
```
In the case of two machine installation, there is no change in the directory structure on machine-1. The directory structure on machine-2 is as follows.
```shell
/opt/autolabjs
    execution_node (please note the singular notation)
    deploy
```

**Comment by prasadtalasila** [comment]
More issues to be taken care of:

1. let the logged in user own /opt/autolabjs directory.
1. let the current user have permission to run the docker containers.
1. provision for five execution nodes in ansible playbooks.

**Comment by AnkshitJain** [comment]
There is an [article](https://docs.docker.com/engine/security/security/#docker-daemon-attack-surface) in the docker documentation, which suggests against having a non-sudo user having permisssions to run docker commands. Nevertheless, if the permissions are granted, a log out from the system is required for the effects to be realized. If we stick with giving the permissions to the user, the log out part will need to be updated in the documentation.

**Comment by prasadtalasila** [comment]
@AnkshitJain you are right. We will stick with the existing setup of requiring sudo access.

**Comment by prasadtalasila** [comment]
Actual builds of the latest commit needs to be done on the integration server for both single machine and two machine scenarios.

**Comment by prasadtalasila** [comment]
On all machines that will have AutolabJS components, the installation directory is `/opt/autolabjs`.

**Comment by prasadtalasila** [comment]
solved in PR #216
